lars-reimann/api-editor
# fix: create boundary type for negative bound

### Summary of Changes

Create boundary types even if one of the bounds was negative.

### Testing Instructions

1. Create API data for scikit-learn
```sh
poetry run parse-package api -p sklearn -o out
```
2. Ensure that boundary types are created even if the bounds are negative
